src/modules/core/components/avatar/avatar.interface.ts

```diff
@@ -0,0 +1,11 @@
+import { type ReactNode } from 'react';
+
+export type AvatarProps = {
+  url?: string;
+  size?: AvatarSize; // default - small
+  fallback?: ReactNode; // default - user icon
+  shadow?: boolean; // default - false
+  shape?: 'circle' | 'rect'; // default - sircle
```

Comment:
I think those comments are unnecessary, please remove them. It creates additional dependency that we don't need, as if you change it in the component you will need to change it here as well. And we can omit it without losing our understanding of the code, so I think that should be fine.